client/landing/jetpack-cloud/sections/scan/main.jsx

```diff
@@ -89,7 +89,25 @@ class ScanPage extends Component {
 	}
 
 	renderScanError() {
-		return <p>There is an error with the scan.</p>;
+		const { siteSlug } = this.props;
+
+		return (
+			<>
+				<SecurityIcon className="scan__icon" icon="scan-error" />
+				<h1 className="scan__header">{ translate( 'Something went wrong' ) }</h1>
+				<p>
+					The scan did not complete successfully. In order to complete the scan you need to contact
```

Comment:
I think we should `translate` this text.

Reply:
I left the translation issue as it is since mostly likely it will change in the future. :S
So I didn't want to translators working on translating something that would most likely change.